Azure.ResourceManager.Core/ResourceGroupOperations.cs

```diff
@@ -9,43 +9,87 @@
 
 namespace Azure.ResourceManager.Core
 {
+    /// <summary>
+    /// A class representing the operations that can be performed over a specific ResourceGroup.
+    /// </summary>
     public class ResourceGroupOperations : ResourceOperationsBase<ResourceGroup>,
         ITaggableResource<ResourceGroup>, IDeletableResource
     {
+        /// <summary>
+        /// Gets initializes a new instance of the <see cref="ResourceType"/> class.
```

Comment:
remove gets and change to ResourceGroupOperations type
```suggestion
        /// Initializes a new instance of the <see cref="ResourceGroupOperations"/> class.
```